python/hail/docs/methods/statgen.rst

```diff
@@ -0,0 +1,50 @@
+Statistical Genetics
```

Comment:
The 'statistical' parts of 'statistical genetics' are in the "statistics" section. I think this should just be 'genetics'. Not worth changing now, though.